src/sherpa_ai/config/task_config.py

```diff
@@ -11,6 +11,7 @@ class AgentConfig(BaseModel):
     verbose: bool = True
     gsite: list[str] = []
     do_reflect: bool = False
+    obsolete: bool = False
```

Comment:
General comment: for anyone who hasn't seen this code before, "obsolete" is too vague.  I suggest calling it something like `use_task_agent`, and in the parser.add_argument and/or elsewhere in the docs you can explain this mode of operation is obsolete.